src/vuex/modules/store-users.coffee

```diff
@@ -10,9 +11,15 @@ mutations =
     state.address = address
   ADD_USER: (state) ->
     state.users.push({
+      id: state.users.length + 1,
```

Comment:
This should be `id: state.id + 1`, when you use `state.users.length` it probably error when you have remove action.